source/common/http/async_client_impl.h

```diff
@@ -195,6 +195,7 @@ class AsyncStreamImpl : public AsyncClient::Stream,
         return std::chrono::milliseconds(0);
       }
     }
+    absl::optional<std::chrono::milliseconds> idleTimeout() const override { return absl::nullopt; }
```

Comment:
Random orthogonal thought that might eventually want to plumb this through for real so that we could have idle timeouts for local origin requests/streams. No action now.